File: 10 - Python/Exercícios/05 - Matrizes/Exercício 04.py
#Exercício - Cada Linha

ordem = int(input("Qual a ordem da matriz? "))

mat = [[0 for x in range(ordem)] for x in range(ordem)]

for i in range(ordem):
    for j in range(0, ordem):
        mat[i][j] = int(input(f"Elemento [{i},{j}]: "))

print("")
print("Maior elemento de cada linha:")

for i in range(ordem):
    maior = [i][0]
    for j in range(ordem):
        if mat[i][j] > maior:
            maior = mat[i][j]
    print(maior)

File: 06 - C/Exercícios/05 - Matrizes/Exercício 04.c
//Exercício - Cada Linha

#include <stdio.h>

int main() {

    int i, j, ordem, maior;

    printf("Qual a ordem da matriz? ");
    scanf("%d", &ordem);

    int mat[ordem][ordem];

    for(i = 0; i < ordem; i++) {
        for(j = 0; j < ordem; j++) {
            printf("Elemento [%d, %d]: ", i, j);
            scanf("%d", &mat[i][j]);
        }
    }

    printf("\nMaior elemento de cada linha:\n");

    for(i = 0; i < ordem; i++) {
        maior = mat[i][0];
        for(j = 1; j < ordem; j++) {
            if(mat[i][j] > maior) {
                maior = mat[i][j];
            }
        }
        printf("%d\n", maior);
    }

    return 0;
}
